src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -9293,7 +9293,7 @@ public class Base
     public string? FalseNMethod() => throw null!; // warn 8
 }
 ";
-            var compilation = CreateCompilation(new[] { source });
+            var compilation = CreateCompilation(new[] { source }, options: WithNonNullTypesTrue());
```

Comment:
>options: WithNonNullTypesTrue() [](start = 66, length = 31)

Is this necessary?

Reply:
With my changes to `VerifyTypes`, it now uses the public API to get type info instead of running the walker manually. Currently, whether or not the nullable analysis is performed is gated on whether the global nullable context is disabled or not. For the tests that use `VerifyTypes` that didn't have this set, I went through and made changes to the ones that I didn't think did this deliberately, and disabled the ones that I thought were relying on the global context being disabled so that we can re-enable later.

---
In reply to: [269210594](https://github.com/dotnet/roslyn/pull/34429#discussion_r269210594) [](ancestors = 269210594)